### ISSUE TITLE
A few questions for training and testing on Caltech

### ISSUE THREAD

**Issue by yijunCai** [amoussawi/caffe#3]
Hi @amoussawi 
Thanks for sharing the code. I'm trying to apply SSD on Caltech too, so I have a few questions here:
1、How do you deal with annotations like 'person?' or 'people' in your trainval set? Also, did you use all ground truth annotations for training, regardless their heights and occlusions? 
2、Since the detection output and evaluation code is for VOC, how do you evaluate your results in Caltech style?
3、Have you tried training only with caltech trainset? I'm interested in how much the performance will boost using extra data(ETH,...) for training.

**Comment by amoussawi** [comment]
1. we ignored `person?` and `people` annotations. We only extract the pedestrians that are at least 20pix tall and at least 20% of their body is visible. This is encoded in `bbSaveSSD` function in `piotr_toolbox/detector/bbGt.m` code.
2. The VOC evaluation code implemented in default SSD outputs a file that contains the detected bounding boxes and their scores. So we parsed that file using `data/caltech/caltech_code/data-USA/res/SSD-ours/parse_det_out.sh` which outputs the detections in the format used by caltech evaluation code.
3. We haven't trained with only caltech trainset due to time limitations. We wanted to get good results asap. We are sure that more data is helpful and that has been shown by many papers. However, we are interested in knowing how much more data helped. We might test it soon.
